src/main/log/Resources/modules/administration/logs/containers/tool.jsx

```diff
@@ -0,0 +1,13 @@
+import {connect} from 'react-redux'
+
+import {withRouter} from '#/main/app/router'
+
+import {LogsTool as LogsToolComponent} from '#/main/log/administration/logs/components/tool'
+
+const LogsTool = withRouter(
+  connect()(LogsToolComponent)
```

Comment:
You connect to to nothing (no props nor actions from store). The connect call is useless and can be removed.
The withRouter is only used to propagate route changes to connected components in some cases. It can also be removed.

You can remove the whole container and directly use the component in this case.